index.js

```diff
@@ -111,6 +111,28 @@ Babel.prototype.optionsHash = function() {
       hash[key] = (typeof value === 'function') ? (value + '') : value;
     }
 
+    if (options.plugins) {
+      hash.plugins = [];
```

Comment:
can we have plugins provide a path to resolve from to find them and their dependents. This would allow babel-transpiler to hash-for-dep them.